component/button/button.css

```diff
@@ -0,0 +1,579 @@
+    color: var(--primary-color) ;
+    }
+  
+  
+  .btn-xs {
+    padding: 0.25rem 0.5rem;
+    font-size: 0.65rem;
+  }
+
+  .btn-sm {
+    padding: 0.4rem 0.75rem;
+    font-size: 0.75rem;
+  }
+  
+  .btn-large{
+    padding: 0.75rem 0.95rem;
+    font-size: 1rem; 
+  }
+
+ .btn-hover:hover{
+    background-color: #f472b6;
+      }
+
+
+.disable{
+    background-color: lightpink;
+}
+
+  
+  .btnn {
+      background-color: var(--secondary-color); 
+      border: none; 
+      color: var(--primary-color); 
+      padding: 14px 16px; 
+      font-size: 16px;
+      cursor: pointer; 
+      
+    }
+
+.link-btn{
+
+padding: 1rem;
+text-decoration: none;
+color: var(--secondary-color);
+
+}
+
+.btn-fab {
+    border-radius: 50%;
+    padding: 1rem 1.15rem;
+    /* Add position fixed when needed */
+}
+
+.btn-fab:hover {
+    box-shadow: 1px 1px 5px var(--primary-color), -1px -1px 5px var(--primary-color);
+}
+
+
+footer{
+    grid-area: footer;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0.5rem;
+    border-top: 1px solid var(--primary-color);
+    margin-top: 3rem;
+      
+
+}
+
+footer ul{
+ padding-inline-start: 0.5rem;
+    display: flex;
+  gap: 20px;
+
+
+} 
+ 
+
+@media (max-width: 1536px){
+
+    html{
+        font-size: 100%;
+    }
+    
+    main{
+        font-size: 1.6rem;
+    }
+
+    iframe{
+        max-inline-size: 100%;
+        height: auto;
+        block-size: auto;
+    }
+    
+ }
+
+
+ 
+ @media (max-width: 1280px){
+
+    html{
+        font-size: 92%;
+    }
+    
+    main{
+        font-size: 1.6rem;
+    }
+
+    iframe{
+        max-inline-size: 100%;
+        height: auto;
+        block-size: auto;
+    }
+    
+ }
+
+
+
+ @media (max-width: 1024px){
+
+    html{
+        font-size: 88%;
+    }
+    
+    main{
+        font-size: 1.6rem;
+         width: 60%;
+    }
+    iframe{
+        max-inline-size: 100%;
+        height: auto;
+        block-size: auto;
+    }
+    
+ }
+
+
+ @media (max-width: 768px){
+
+    html{
+       font-size: 84%;
+    }
+
+    nav ul{
+        padding: 2.8rem;
+        margin: auto;
+         }
+     
+     
+      .component-text{
+          width: 40%;
+          text-align: left;
+          font-size: 1.6rem;
+      }
+      
+      hr{
+         display: none;
+        
+      }
+     
+       main{
+         position: relative;
+         left: -16rem;
+         font-size: 1.6rem;
+         box-sizing: border-box;
+         padding: 2.4em;
+         width: 140%;
+       }
+     
+          aside{
+              box-sizing: border-box;
+              position: absolute;
+              top: 16rem;
+              z-index: 2;
+        
+              
+              
+     
+          }
+         
+     aside ul{
+         background-color: var(--primary-color);
+     }
+     
+          aside ul li a{
+              color: var(--black);
+          }
+     
+          .hamnav{

```

Comment:
follow a specific naming convention to name your classnames so that its easier for you to debug and the reviewer to understand what is the target to which the style is being applied to